api/src/main/java/org/openmrs/module/ugandaemrreports/library/HIVPatientDataLibrary.java

```diff
@@ -446,6 +456,10 @@ public PatientDataDefinition getTransferInRegimen() {
     public PatientDataDefinition getArtStartDate() {
         return df.getObs(hivMetadata.getArtStartDate(), Arrays.asList(hivMetadata.getARTSummaryEncounter()), TimeQualifier.FIRST, df.getObsDatetimeConverter());
     }
+    public PatientDataDefinition getViralLoadDate() {
```

Comment:
@solemabrothers Call this getMostRecentViralLoadDate() to convey just in case we need getFirstViralLoadDate()

Reply:
This has been fixed